load-test-framework/src/main/java/com/google/pubsub/flic/controllers/Client.java

```diff
@@ -111,6 +111,7 @@ public static String getTopicSuffix(ClientType clientType) {
       case CPS_GCLOUD_JAVA_SUBSCRIBER:
       case CPS_GCLOUD_PYTHON_PUBLISHER:
       case CPS_GCLOUD_GO_PUBLISHER:
+      case CPS_GCLOUD_GO_SUBSCRIBER:
```

Comment:
Since you have a publisher and subscriber you should have a new prefix, that way you can run a test comparing go e2e against Java e2e. You could use gcloud-go